browser/css/toolbar.css

```diff
@@ -1154,7 +1154,7 @@ button.leaflet-control-search-next
 .w2ui-icon.help { background: url('images/lc_helpindex.svg') no-repeat center; }
 .w2ui-icon.insertpage { background: url('images/lc_insertpage.svg') no-repeat center; }
 .w2ui-icon.conditionalformatdialog { background: url('images/lc_conditionalformatmenu.svg') no-repeat center; }
-.w2ui-icon.search { background: url('images/sc_recsearch.svg') no-repeat center; }
+.w2ui-icon.search { background: url('images/lc_recsearch.svg') no-repeat center; }
 .w2ui-icon.presentation { background: url('images/lc_dia.svg') no-repeat center; }
 .w2ui-icon.sign_ok { background: url('images/sign_ok.svg') no-repeat center; }
```

Comment:
we use sc for icons that were designed for smaller sizes than 24 , in this case 16px. So possibly better would be to redesign the sc_recsearch.svg unless the image is displayed on mobile and set as 24px